docs/using-hive.md

```diff
@@ -77,87 +234,77 @@ metadata:
 spec:
   baseDomain: hive.example.com
   clusterName: mycluster
-  compute:
-  - name: worker
-    platform:
-      aws:
-        rootVolume:
-          iops: 100
-          size: 22
-          type: gp2
-        type: m4.large
-    replicas: 3
-  controlPlane:
-    name: master
-    platform:
-      aws:
-        rootVolume:
-          iops: 100
-          size: 22
-          type: gp2
-        type: m4.large
-    replicas: 3
-  imageSet:
-    name: openshift-v4.2.0
-  images:
-    installerImagePullPolicy: Always
-  networking:
-    clusterNetworks:
-    - cidr: 10.128.0.0/14
-      hostSubnetLength: 23
-    machineCIDR: 10.0.0.0/16
-    serviceCIDR: 172.30.0.0/16
-    type: OpenShiftSDN
+  controlPlaneConfig:
```

Comment:
I would omit this unless you are going to give examples of using serving certs.

Reply:
done